src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -236,8 +248,7 @@ private void InitializeCore()
 
         private void LibraryRootOnPropertyChanged(object sender, PropertyChangedEventArgs propertyChangedEventArgs)
         {
-            if (propertyChangedEventArgs.PropertyName == "Visibility")
```

Comment:
When we change workspace(i.e. make Input and Output nodes visible) we shouldn't start search. But we can use this PropertyChanged event to hide top-result tooltip.